### PR TITLE
#5140 [Manager] Creates a component to appear count of re-render

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5140 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
> Do we have a follow-up to apply this powerful tool to other components?

I think this is only for testing how many render counts a given component has. Maybe we could add an example link in the comment?

File: ohara-manager/client/src/components/common/RenderCount.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ *    <RenderCount />
+ *    <Form>
+ *      <Field>
+ *        <RenderCount />
+ *        <label>First Name</label>
+ *        <input name="firstName" placeholder="First Name" />
+ *      </Field>
+ *    </Form>
+ * </Dialog>
+ */
```
Comment (eechih) [inline ohara-manager/client/src/components/common/RenderCount.js:35]: @oharastream/frontend Explain the purpose and usage of this component

File: ohara-manager/client/src/components/common/RenderCount.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+/**
+ * The purpose of this component is to help developers avoid to writing redundant * and invalid rendering. It is recommended to use it only in the development.
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/RenderCount.js:21]: the format is a bit off...

Comment (eechih) [reply ohara-manager/client/src/components/common/RenderCount.js]: I will fix it :)

**Pull request merged**